vm-setup/roles/v1aX_integration_test/tasks/node_reuse.yml

```diff
@@ -179,7 +132,7 @@
         bmhs.resources | filter_provisioning("available,ready") | map(attribute='metadata.name')
         )
 
-  - name: Check if the deprovisioned BMH is re-used for the next provisioning.
+  - name: Check node reuse logic ensuring the just deprovisioned BMH is re-used for the next provisioning.
```

Comment:
nit: there is seems to be a typo in this text

Reply:
as is also should be clear, but if you think not, I can change it, wanted to avoid re-running feature tests for name change, let me know your opinion

Reply:
OK sure, "Check node reuse logic ensuring the just deprovisioned" part looked a bit odd to me. But that's not something to worry, let's move on

Reply:
Ok, I have fixed it now, should be good now

Reply:
This looked cleaner. Thanks